dockerfiles/cli/scripts/entrypoint.sh

```diff
@@ -12,7 +12,7 @@
 CHE_PRODUCT_NAME="CODENVY"
 CHE_MINI_PRODUCT_NAME="codenvy"
 CHE_FORMAL_PRODUCT_NAME="Codenvy"
-CHE_CONTAINER_ROOT="/codenvy"
+CHE_CONTAINER_ROOT="/data"
```

Comment:
Found the issue and fixed it in the /init templates. The /init image has to be rebuilt with these changes.